### PR TITLE
2018 Lexus RX 350 fingerprint

### PR DESCRIPTION
**Pull request by Afshin1354**
Add FINGERPRINT and carFw for 2018 LEXUS RX 350.

### CODE REVIEW

File: selfdrive/car/toyota/values.py [hunk 1]
```diff
@@ -114,6 +114,10 @@ class CAR:
...
+  },
+  # 2018 Lexus RX 350
+  {
+    36: 8, 37: 8, 114: 5, 119: 6, 120: 4, 170: 8, 180: 8, 186: 4, 426: 6, 452: 8, 464: 8, 466: 8, 467: 8, 544: 4, 550: 8, 552: 4, 562: 6, 608: 8, 610: 5, 643: 7, 658: 8, 705: 8, 740: 5, 742: 8, 743: 8, 800: 8, 810: 2, 812: 3, 814: 8, 830: 7, 835: 8, 836: 8, 869: 7, 870: 7, 871: 2, 898: 8, 900: 6, 902: 6, 905: 8, 918: 8, 921: 8, 933: 8, 944: 8, 945: 8, 951: 8, 955: 8, 956: 8, 998: 5, 999: 7, 1000: 8, 1001: 8, 1005: 2, 1014: 8, 1017: 8, 1020: 8, 1041: 8, 1042: 8, 1044: 8, 1056: 8, 1059: 1, 1063: 8, 1077: 8, 1082: 8, 1114: 8, 1161: 8, 1162: 8, 1163: 8, 1164: 8, 1165: 8, 1166: 8, 1167: 8, 1227: 8, 1228: 8, 1235: 8, 1237: 8, 1264: 8, 1279: 8, 1552: 8, 1553: 8, 1556: 8, 1557: 8, 1568: 8, 1570: 8, 1571: 8, 1572: 8, 1575: 8, 1595: 8, 1777: 8, 1779: 8, 1904: 8, 1990: 8, 1998: 8
```
Comment (adeebshihadeh) [inline selfdrive/car/toyota/values.py:120]: merge this with the 2017 fingerprint

Comment (pd0wm) [reply selfdrive/car/toyota/values.py:120]: We shouldn't be merging new fingerprints for Honda/Toyota. The risk of breaking existing cars is too big.

Comment (Afshin1354) [reply selfdrive/car/toyota/values.py:120]: > merge this with the 2017 fingerprint

Hi @adeebshihadeh. Would you please elaborate on your suggestion? By merge you men combine 2017 and 2018 into one dictionary?

**Comment by VirtuallyChris** [comment]
Please remove the fingerprint 1.0 output and just add the car engine firmware version, then we can merge @Afshin1354

**Comment by VirtuallyChris** [comment]
If the car uses fingerprinting 2.0, it ignores the 1.0 value. So the 1.0 doesn't matter anymore 👍

**Pull request merged**